images/win/scripts/SoftwareReport/SoftwareReport.Android.psm1

```diff
@@ -156,3 +157,13 @@ function Get-AndroidGoogleAPIsVersions {
     return ($versions -Join "<br>")
 }
 
+function Build-AndroidEnvironmentTable {
+    $androidVersions = Get-Item env:ANDROID_*	
+
+    return $androidVersions | ForEach-Object {	
+        [PSCustomObject] @{	
+            "Name" = $_.Name	
+            "Value" = Get-Path-With-Link($_.Value)
```

Comment:
I don't think that it makes sense to print link in case when it points to `C:\Program Files`:
```
 ----------------------- | ---------------------------------------------------------------------------- |
| ANDROID_SDK_ROOT        | C:\Android\android-sdk -> C:\Program Files (x86)\Android\android-sdk         |
| ANDROID_NDK_LATEST_HOME | C:\Android\android-sdk\ndk\22.0.7026061                                      |
| ANDROID_NDK_PATH        | C:\Android\android-sdk\ndk-bundle -> C:\Android\android-sdk\ndk\21.4.7075529 |
| ANDROID_HOME            | C:\Android\android-sdk -> C:\Program Files (x86)\Android\android-sdk         |
| ANDROID_NDK_ROOT        | C:\Android\android-sdk\ndk-bundle -> C:\Android\android-sdk\ndk\21.4.7075529 |
| ANDROID_NDK_HOME        | C:\Android\android-sdk\ndk-bundle -> C:\Android\android-sdk\ndk\21.4.7075529 
```

In my opinion it could confuse customers. Actually, we have the single symlink `ndk-bundle` and it makes sense to resolve only it.